src/components/months-slider.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from "prop-types"
+import { useCallback } from "react";
+import { Slider } from "@mui/material";
+
+const valueLabelFormatByMonths = (totalMonths) => (value) => {
+    const diff = totalMonths - value;
+    const date = new Date()
+    date.setMonth(date.getMonth() - diff)
+    return `${date.getMonth() + 1}/${date.getFullYear()}`;
```

Comment:
there is an off-by-one bug here, fixed in 5ee580a

Reply:
Lo aplico manualmente en mi rama, gracias por notarlo

Reply:
Solucionado